packages/react/src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -81,7 +81,7 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, any> = {
       alignItems: 'center',
       position: 'relative',
       padding: `0 ${pxToRem(paddingLeftRightValue)}`,
-      margin: `0 ${pxToRem(8)} 0 0`,
+      margin: 0,
```

Comment:
here is where we remove redundant margin for `Icon`

Reply:
could we omit this line altogether?

Reply:
done, thanks